pyanaconda/ui/lib/subscription.py

```diff
@@ -145,6 +143,17 @@ def noop(*args, **kwargs):
     pass
 
 
+def dummy_error_callback(error):
```

Comment:
Is this used somewhere?

Reply:
I don't think its used anymore - this was used by an older revision of the code, but is not used anymore. So removed, thanks! :)